### ISSUE TITLE
Error while starting electron

### ISSUE THREAD

**Issue by ghost** [dolanmiu/ng-cli-angular#1]
While __"nge serve"__ & __"nge build"__ commands are run then it must be checked if __"main" : "main.js"__
is present in the package.json file, if not then add it(or change the generated file-name from __"main.js"__ to __"index.js"__).

**Comment by ghost** [comment]
This took me a while to figure out as I'm a newbie in __electron__ &  __node.js__ . BTW thanks for this plugin/package. :smile:

**Comment by dolanmiu** [comment]
hmm, this package doesn't check the `"main" : "main.js"` thing you mentioned. It's quite basic actually 😂, just make sure you have this in your scripts:

```
"scripts": {
    "cool": "nge build",
}
```

It should generate the `main.js` for you (I'm guessing you are referring to the Electron main, not the Angular one). You shouldn't need to have your own personal `main.js`, if so, then it's a bug!

**Comment by ghost** [comment]
Yeah I get that it should automatically generate the __main.js__ file. This is what I did :-

```
ng new some-random-app
cd some-random-app
npm install -g ng-cli-electron // installed globally in case I need to use it on multiple projects
```
 Then added the following to the __scripts__ in package.json
```
"build-electron": "nge build",
"serve-electron": "nge serve"
```
Now after executing __nge serve__, an error is displayed saying that an electron app was not found in __...blah/...blah/dist__ folder [can't remember the exact error message] :smirk:

I checked the electron documentation & found out that there either needs to be a __{ "main" : "main.js" }__ line in the __package.json__ file(in case __main.js__ is the main __Electron__ file) or it should be omitted in case the main __Electron__ file is __index.js__. I think you get my point now. Or maybe this isn't the way how the extension/plugin was supposed to be used?

**Comment by dolanmiu** [comment]
I see!! 

thanks for this, I add the `main` without thinking, becomes habit, but I agree its not immediately obvious. Will be released as an update ASAP

**Comment by dolanmiu** [comment]
Fixed in `1.0.3`
